app/src/main/res/layout/tx_details_rejection.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:animateLayoutChanges="true"
+    android:background="@color/surface_01"
+    android:orientation="vertical">
+
+    <TextView
+        android:id="@+id/tx_rejection_info"
+        style="@style/TextDark.Bold"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="@dimen/default_margin"
+        android:layout_marginRight="@dimen/default_margin"
+        android:layout_marginTop="@dimen/default_margin"
+        tools:text="@string/tx_details_rejection_info" />
+
+    <TextView
+        android:id="@+id/tx_payment_reason_link"
+        style="@style/TextLink"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/default_margin"
+        android:layout_marginLeft="@dimen/default_margin"
+        android:layout_marginRight="@dimen/default_margin"
+        android:layout_marginTop="10dp"
+        android:drawablePadding="6dp"
+        android:gravity="center_vertical"
+        android:text="@string/tx_details_rejection_payment_reason"
+        app:drawableEndCompat="@drawable/ic_external_link_green_16dp" />
```

Comment:
This doesn't place the icon at the end of the text. When I had to create links with icons like this, I used `TextView.appendLink()` like in the following example:

```
sectionTitle.appendLink(
                url = resources.getString(R.string.tx_list_conflict_header_link),
                urlText = resources.getString(R.string.tx_list_conflict_header_learn_more),
                linkIcon = R.drawable.ic_link_green_24dp,
                prefix = " "
            )
```

Reply:
we don't need it here. Whole text is clickable. This is how it looks
![image](https://user-images.githubusercontent.com/17750984/108222752-59e22700-7139-11eb-84fb-b900d11c9983.png)

Reply:
On my phone it looked like this: 
![image](https://user-images.githubusercontent.com/246473/108224850-a0388580-713b-11eb-9a88-55af9275d810.png)